src/input/hotkey.c

```diff
@@ -400,6 +400,6 @@ void hotkey_func(int f_number, int with_modifier)
         case 7: system_resize(640, 480); break;
         case 8: system_resize(800, 600); break;
         case 9: system_resize(1024, 768); break;
-        case 12: take_screenshot(); break;
+        case 12: take_screenshot(with_modifier); break;
```

Comment:
`with_modifier` includes every key (shift, alt, control). This does not work properly on Mac, since F12 is a system key, so I have to press at least one of those to be able to send F12. I can only save full city screenshots now.
Suggestion: add `with_ctrl` as extra parameter and use that.

Reply:
On original C3, what's supposed to be the modifier hotkey for bookmarks?

I ask because I think a function with the params `with_modifier, with_ctrl` doesn't look good, might as well change it `with_shift<or alt for bookmarks?>, with_ctrl`.